### ISSUE TITLE
Reply by Email Review and Refactoring

### ISSUE THREAD

**Issue by namangupta01** [publiclab/plots2#8356]
Hey! I was recently using reply by email feature. There was no hide and show button for quoted content in comment as we used to have at the starting.
As in the image, quoted content is visible and we don't have any show and hide button for quoted content.
<img width="1440" alt="Screenshot 2020-08-27 at 01 02 25" src="https://user-images.githubusercontent.com/20709985/91348456-69735980-e801-11ea-8ce0-56f21eaf8dc8.png">

@jywarren Did we intentionally removed it? If not, then I would like to fix it.

And also @jywarren did we have any issue with the email parsing in the past? 
Do you have any idea why we are using ?https://github.com/publiclab/plots2/blob/ce68627ccb452f2aed1f536fa6ad7fd01d31e57c/app/models/comment.rb#L473

Thank You!!

**Comment by ebarry** [comment]
Thanks for reporting this @namangupta01 ! also, it's great to hear from you 💡
I have also noticed this over the past few months and was concerned. It is not our intention to show the quoted content, but you'd have to ask @jywarren or @cesswairimu perhaps for details

**Comment by ebarry** [comment]
Thanks again for spotting this @namangupta01 , I think several active users have pointed out the same issue with email signatures showing up in comments posted by email (again).

**Comment by cesswairimu** [comment]
I don't think we removed it linking https://github.com/publiclab/plots2/issues/7321 which is related. @namangupta01 we would love your help.

**Comment by namangupta01** [comment]
Yeah sure! I will work on it.

**Comment by jywarren** [comment]
omg that would be great! Thanks @namangupta01 and so glad to see you! Perhaps we can add a small system test assertion for that to protect it as well.

**Comment by aliciapaz** [comment]
Hey, @TildaDares I'd like to work on this. 

I was trying to reproduce the error but I didn't know how really (since it's an email feature)

**Comment by TildaDares** [comment]
Hi @aliciapaz, [Mailman](https://github.com/mailman/mailman) is the gem used for the `reply by email` feature but it's hard to get any recent/helpful documentation on how to use it. 

I tried running the command in [our documentation](https://github.com/publiclab/plots2/blob/main/doc/EMAIL.md#how-to-start-reply-by-email-to-comment-feature) for the feature but I'm getting this error

```
I, [2022-01-28T02:45:14.980283 #52855]  INFO -- : Mailman v0.7.0 started
I, [2022-01-28T02:45:14.980375 #52855]  INFO -- : Rails root found in ., requiring environment...
I, [2022-01-28T02:45:24.186264 #52855]  INFO -- : POP3 receiver enabled (@).
I, [2022-01-28T02:45:24.205482 #52855]  INFO -- : Polling enabled. Checking every 20 seconds.
E, [2022-01-28T02:45:24.206249 #52855] ERROR -- : Connection refused - connect(2) for nil port 995
```

This is the code for the `reply by email`
https://github.com/publiclab/plots2/blob/b11aa99180db020562a9e379e63081f4e31995f3/app/models/comment.rb#L239-L262

**Comment by TildaDares** [comment]
@jywarren Since [Action Mailbox](https://guides.rubyonrails.org/action_mailbox_basics.html) has more documentation and support, can we think of switching to it instead of using mailman?

**Comment by jywarren** [comment]
Gosh this has been broken for some time. I have been wanting to try debugging with Sebastian but we have been doing a series of big migrations of other systems to Google Cloud. I want to say we should be able to get mailman working again but it's a good point about Action Mailbox being more supported. How big of a project do you think the transition would be?

**Comment by Forchapeatl** [comment]
Please may I work on this issue ?

**Comment by jywarren** [comment]
Hi @Forchapeatl I worry this is a really gigantic project -- say, "switching to Action Mailbox" for example, or even the kind of deep digging it might take to debug mailman, since we use multiple Docker containers to run this. Does some part of it look like a smaller piece you could try taking on?
